dapps/onboarding.vue/src/components/mnemonic/mnemonic.ts

```diff
@@ -149,14 +149,13 @@ export default class Mnemonic extends mixins(EvanComponent) {
 
   async getFilteredMnemonicWords(index) {
     if (!this.words[index] || this.words[index].length < 2) {
-      return [ ];
-    } else {
-      const lowerCase = this.words[index].toLowerCase();
-
-      return this.lightWalletWords
-        .filter(word => word.indexOf(lowerCase) === 0 && word !== lowerCase)
-        .slice(0, 10);
+      return [];
     }
+    const lowerCase = this.words[index].toLowerCase();
+
+    return this.lightWalletWords
```

Comment:
what is this actually used for? Looks bit like autocompletion, but we don't do this anymore right?

Reply:
Maybe it needs a refactoring, its quite old code.